src/components/organize/tasks/TaskStatusChip.tsx

```diff
@@ -8,6 +8,7 @@ enum ChipColors {
   closed = '#FF4242',
   draft = 'grey',
   expired = 'black',
+  passed = '#dac82f',
```

Comment:
Can remove `closed`. Or just rename `closed` to `passed` and not add a new colour.